Automattic/wp-calypso
# Update/jetpack cloud folder structure

#### Changes proposed in this Pull Request

* Code clean up. Moves the backup and scan sections into a single section. 
* This will make it easier to implement sections for jetpack cloud since it follows more of the calypso (my-sites) pattern where the sub pages of a sections are places inside the same folder. 

I also removed the verbose jetpackCloud prefixes. Since they are not necessary and it makes the code easier to read. 

#### Testing instructions
* The testing instructions are the same as https://github.com/Automattic/wp-calypso/pull/39584

Visit the following pages, with your own jetpack test site slug substituted. Each should load its respective page with correctly resolved blog ID (and in some cases, download/backup/restore ID).
```
    jetpack.cloud.localhost:3000/dashboard
    jetpack.cloud.localhost:3000/backups/{siteSlug}
    jetpack.cloud.localhost:3000/backups/{siteSlug}/details/12345678
    jetpack.cloud.localhost:3000/backups/{siteSlug}/download/12345678
    jetpack.cloud.localhost:3000/backups/{siteSlug}/restore/123455678
    jetpack.cloud.localhost:3000/scan/{siteSlug}
    jetpack.cloud.localhost:3000/scan/{siteSlug}/history
    jetpack.cloud.localhost:3000/settings/{siteSlug}
```
